doc/api/stream.md

```diff
@@ -253,16 +469,16 @@ readable: null
 end
 ```
 
-#### readable.isPaused()
+##### readable.isPaused()
 
 * Return: {Boolean}
 
-This method returns whether or not the `readable` has been **explicitly**
-paused by client code (using [`stream.pause()`][stream-pause] without a
-corresponding [`stream.resume()`][stream-resume]).
+The `readable.isPaused()` method returns `true` if `readable` has been
+**explicitly** paused by client code (using the [`stream.pause()`][stream-pause]
+method without a corresponding call to [`stream.resume()`][stream-resume]).
```

Comment:
Oof. This method was (my) mistake. The weird bit about this method is that it'll say that fresh streams aren't paused (since they're in that "flowing=null" state). We should probably note that it exists, and that it (weirdly) reports some paused streams as not-really-paused, but recommend that folks avoid it.